ui/src/AccountDropdown/AccountDropdown.test.tsx

```diff
@@ -25,6 +25,7 @@ import {MemoryHistory} from 'history/createMemoryHistory';
 import SpaceClient from '../Space/SpaceClient';
 import Cookies from 'universal-cookie';
 import {RunConfig} from '../index';
+import {GlobalStateProps} from "../Redux/Reducers";
```

Comment:
Please run `npm run lint:fix` to fix double quotes and such.

Reply:
Yeah i need to update my auto import not to do that, thanks for the tip on the lint command